Patches/Android Tiers/Hediffs/Hediffs_Android.xml

```diff
@@ -142,10 +143,10 @@
 					<capacities>
 						<li>Bite</li>
 					</capacities>
-					<power>31</power>
-					<cooldownTime>1.86</cooldownTime>
-					<armorPenetrationSharp>4.22</armorPenetrationSharp>
-					<armorPenetrationBlunt>13.5</armorPenetrationBlunt>
+					<power>44</power>
+					<cooldownTime>2.51</cooldownTime>
+					<armorPenetrationSharp>14.07</armorPenetrationSharp>

```

Comment:
What creatures are these jaws for? They're extremely strong (for comparison, the Grizzly bear has the strongest vanilla Bite with 31 damage) and the Sharp AP may be too high.